docs/gaia/delegator-guide-cli.md

```diff
@@ -0,0 +1,405 @@
+
+1. Download the Ledger Live app [here](https://www.ledger.com/pages/ledger-live)
+2. Connect your ledger via USB and update to the latest firmware
+3. Go to the ledger live app store, and download the "Cosmos" application (this can take a while)
+4. Navigate to the Cosmos app on your ledger device
+
+Then, to create an account, use the following command:
+
+```bash
+gaiacli keys add <yourAccountName> --ledger 
+```
+
+- `<yourKeyName>` is the name of the account. It is a reference to the account number used to derive the key pair from the mnemonic. You will use this name to identify your account when you want to send a transaction.
+- You can add the optional `--account` flag to specify the path (`0`, `1`, `2`, ...) you want to use to generate your account. By default, account `0` is generated. 
+
+#### Using a computer 
+
+::: warning
+**NOTE: It is more secure to perform this action on an offline computer**
+:::
+
+To generate an account, just use the following command:
+
+```bash
+gaiacli keys add <yourKeyName>
+```
+
+The command will generate a 24-words mnemonic and save the private and public keys for account `0` at the same time. You will be prompted to input a passphrase that is used to encrypt the private key of account `0` on disk. Each time you want to send a transaction, this password will be required. If you lose the password, you can always recover the private key with the mnemonic. 
+
+::: danger
+**NEVER LOSE YOUR MNEMONIC. WRITE IT DOWN ON A PIECE OF PAPER AND STORE IT SOMEWHERE SAFE. IF YOU LOSE IT, THERE IS NO WAY TO RETRIEVE IT. IF SOMEONE GAINS ACCESS TO IT, THEY GAIN ACCESS TO ALL THE ASSOCIATED ACCOUNTS**
+::: 
+
+::: warning
+After you have secured your mnemonic (triple check!), you can delete bash history to ensure no one can retrieve it:
+
+```bash
+history -c
+rm ~/.bash_history
+```
+:::
+
+- `<yourKeyName>` is the name of the account. It is a reference to the account number used to derive the key pair from the mnemonic. You will use this name to identify your account when you want to send a transaction.
+- You can add the optional `--account` flag to specify the path (`0`, `1`, `2`, ...) you want to use to generate your account. By default, account `0` is generated. 
+
+
+You can generate more accounts from the same mnemonic using the following command:
+
+```bash
+gaiacli keys add <yourKeyName> --recover --account 1
+```
+
+This command will prompt you to input a passphrase as well as your mnemonic. Change the account number to generate a different account. 
+
+
+## Accessing the Cosmos Hub network
+
+In order to query the state and send transactions, you need a way to access the network. To do so, you can either run your own full-node, or connect to someone else's.
+
+::: warning
+**NOTE: Never give your mnemonic, private key or passphrase to anyone. If a full-node operator requests it, they are trying to abuse you**.
```

Comment:
It may be better to say, "Do not share your 12 words with anyone. The only person who should ever need to know them is you. This is especially important if you’re ever approached via email or direct message by someone requesting that you share your mnemonic for any kind of blockchain services or support. No one from Cosmos, the Tendermint team or the Interchain Foundation will ever send an email that asks for you to share any kind of account credentials or your 12 words."